alg-excer1-palindrome.js

```diff
@@ -0,0 +1,15 @@
+function palindrome(input) {
+  str =  input.toString().toLowerCase();
+  var strLength= str.length;
+
+  for (var i=0; i<strLength/2; i++) {
+    if (str[i] !== str[strLength-1-i]) {
+      return input + "- is not Palindrome";
+    }
+  }
+  return input + "- is Palindrome";
```

Comment:
stop adding text to your results!!!! how do you expect to reuse this function if you keep adding text to it? What should it return? What makes more sense?

Reply:
updated